frontend/public/locales/en/translation.json

```diff
@@ -25,6 +25,10 @@
   "deleteUserBtn": "Delete User",
   "edit": "Edit",
   "editUserDetails": "Edit user details",
+  "removeParticipant": "Are you sure you wish to remove the user from the loop?",
+  "remove": "remove",
+  "demote": "Are you sure you would like to revoke host privelleges from the user?",
```

Comment:
Add the users name as a variable using double curly brackets, instead of `user`

See purge user translation for reference

Reply:
See 68f9d19d6d77683950d275a1be2563d0950ff601